src/Evolution/Systems/Cce/OptionTags.hpp

```diff
@@ -497,5 +506,20 @@ struct InitializeJ : db::SimpleTag {
   }
 };
 
+/// A tag that constructs a `AnalyticBoundaryDataManager` from options
+struct AnalyticBoundaryDataManager : db::SimpleTag {
+  using type = ::Cce::AnalyticBoundaryDataManager;
+  using option_tags = tmpl::list<OptionTags::ExtractionRadius, OptionTags::LMax,
+                                 OptionTags::AnalyticSolution>;
+
+  static constexpr bool pass_metavariables = false;
+  static Cce::AnalyticBoundaryDataManager create_from_options(
+      const double extraction_radius, const size_t l_max,
+      const std::unique_ptr<Cce::Solutions::WorldtubeData>&
+          worldtube_data) noexcept {
+    return ::Cce::AnalyticBoundaryDataManager(l_max, extraction_radius,
+                                            worldtube_data->get_clone());
```

Comment:
this formatting looks off. Perhaps just run `git-clang-format upstream/develop` and add the output as a separate commit?